specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -11015,7 +11019,7 @@
         "jobAction": {
           "type": "string",
           "title": "An action to take on the job containing the task, if the task completes with the given exit condition and the job's onTaskFailed property is 'performExitOptionsJobAction'.",
-          "description": "The default is none for exit code 0 and terminate for all other exit conditions. If the job's onTaskFailed property is noAction, then specify this property returns an error. The add task request fails with an invalid property value error;; if you are calling the REST API directly, the HTTP status code is 400 (Bad Request).",
+          "description": "Values are: none - take no action. disable - disable the job. This is equivalent to calling the disable job API, with a disableTasks value of requeue. terminate - terminate the job. The terminateReason in the job's executionInfo is set to \"TaskFailed\". The default is none for exit code 0 and terminate for all other exit conditions. If the job's onTaskFailed property is noAction, then specifying this property returns an error and the add task request fails with an invalid property value error; if you are calling the REST API directly, the HTTP status code is 400 (Bad Request).",
```

Comment:
>disable - disable the job. This is equivalent to calling the disable job API, with a disableTasks value of requeue. terminate [](start = 61, length = 125)

maybe there is no better option with the present d.m.c/AutoRest situation but having a sentence after one enum value and then going straight into the name of the next is really hard to read.  I don't know if embedded newlines or any other formatting will save us - if not perhaps we need something like:

"none - ....; disable - disables the job; terminate - terminates the job.  The disable option is equivalent to...  In the terminate option, the terminateReason..."

Still gets messy though.  We need them to fix this paragraphing issue!